server/embed/config.go

```diff
@@ -205,6 +207,10 @@ type Config struct {
 	MaxTxnOps           uint   `json:"max-txn-ops"`
 	MaxRequestBytes     uint   `json:"max-request-bytes"`
 
+	// MaxConcurrentStreams specifies the maximum number of concurrent
+	// streams that a client can open at a time.
```

Comment:
```suggestion
	// MaxConcurrentStreams specifies the maximum number of concurrent
	// streams that each client may have open at a time. 
```

Reply:
Anyway, I replaced the `a` with `each`, and kept the `can` as it's, just as I mentioned above. This is the best I can tell from my perspective as a non-native English speaker :)

Reply:
:) thanks much @ahrtr I added couple of nits related to it, but it looks good to me. As long as it's technically correct we are good.